seed_database/seedPostgreSQL.js

```diff
@@ -0,0 +1,61 @@
+const { Pool } = require('pg');
+const path = require('path');
+const pool = new Pool({
+  user: 'postgres',
+  host: '127.0.0.1',
+  database: 'sdcandrew',
+  password: '',
+  port: 5432,
+});
+
+const seedPostgres = async () => {
+  const client = await pool.connect();
```

Comment:
if `pool.connect()` can be `await`ed, maybe put this in the `try`-`catch` block as well?

Reply:
I tried this method previously but had problems using the client to perform release or rollback queries within the catch block and finally block!

Most examples within the documentation also created the pool connection outside the try blocks but ended the pool connection within the blocks.
https://node-postgres.com/features/pooling